docs/source/extensions/logging.rst

```diff
@@ -130,19 +130,14 @@ The :meth:`~pytorch_lightning.core.lightning.LightningModule.log` method has a f
 * ``on_step``: Logs the metric at the current step.
 * ``on_epoch``: Automatically accumulates and logs at the end of the epoch.
 * ``prog_bar``: Logs to the progress bar (Default: ``False``).
-* ``logger``: Logs to the logger like ``Tensorboard``, or any other custom logger passed to the
-    :class:`~pytorch_lightning.trainer.trainer.Trainer` (Default: ``True``).
+* ``logger``: Logs to the logger like ``Tensorboard``, or any other custom logger passed to the :class:`~pytorch_lightning.trainer.trainer.Trainer` (Default: ``True``).
```

Comment:
The changes here fix a weird issue where some text appears in a larger font.

Before: https://pytorch-lightning.readthedocs.io/en/latest/extensions/logging.html#automatic-logging

After: https://152279-178626720-gh.circle-artifacts.com/0/html/extensions/logging.html#automatic-logging

Reply:
FYI, this was because the indentation whitespace wasn't the right amount. It has to align properly with the start of the previous line after the bullet point indentation.

Reply:
I see - it looks like it's fixed now though right?